src/main/java/uk/gov/hmcts/reform/sendletter/services/date/holidays/response/Holidays.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.sendletter.services.date.holidays.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class Holidays {
+    public final String division;
```

Comment:
Division doesn't seem to be needed as data, given that we know which one we want to get data for

Reply:
removed